src/azure/index.ts

```diff
@@ -0,0 +1,23 @@
+import { AzureFunction, Context } from "@azure/functions";
+
+import { Handler } from "..";
+import { HttpResponse } from "../http";
+
+export * from "./http";
+export * from "./queue";
+export * from "./blob";
+
+export const unsafeRun =
+  (handler: Handler<Context>): AzureFunction =>
+  async (ctx) => {
+    const output = await handler(ctx)();
```

Comment:
what if handler throws here?

Reply:
The Azure Function crashes with that error, such as when "Error" is returned as output